projects/packages/videopress/src/class-block-editor-content.php

```diff
@@ -21,36 +21,200 @@ public static function init() {
 			return;
 		}
 
-		add_filter( 'default_content', array( static::class, 'video_block_by_guid' ), 10, 2 );
+		global $shortcode_tags;
+
+		// Remove the videopress shortcodes added by the Jetpack plugin.
+		if ( array_key_exists( 'videopress', $shortcode_tags ) ) {
```

Comment:
You can also do `shortcode_exists( 'videopress' )` ([ref](https://github.com/WordPress/WordPress/blob/master/wp-includes/shortcodes.php#L133)).

Reply:
Nice! I'll change that, thanks!